libraries/resource_providers/api600/c7000/server_certificate_provider.rb

```diff
@@ -0,0 +1,68 @@
+# (c) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+module OneviewCookbook
+  module API600
+    module C7000
+      # ServerCertificate API600 C7000 provider
+      class ServerCertificateProvider < ResourceProvider
+        def import_certificate
+          @item.data['remoteIp'] = @item.data['name']
+          x = @item.get_certificate
+          @item.data.clear
+          @item.data['type'] = x['type']
+          @item.data['certificateDetails'] = []
+          @item.data['certificateDetails'][0] = {
+            'type' => x['certificateDetails'][0]['type'],
+            'base64Data' => x['certificateDetails'][0]['base64Data']
+          }
+          @item.data['name'] = @item.data['remoteIp']
+          @item.data['aliasName'] = @item.data['remoteIp']
+          @item.data['uri'] = "/rest/certificates/servers/#{@item.data['aliasName']}"
+          @item.data.delete('remoteIp')
+          Chef::Log.info 'SSL_CERTIFCATE IMPORTED SUCCESSFULLY'
```

Comment:
this message should be printed after importing

Reply:
@soodpr Corrected